### PR TITLE
Handle change event - Change language

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: changes/tests.py [hunk 1]
```diff
@@ -696,3 +696,145 @@ def test_friend_only_change_baby(self):
...
+                }],
+            },
+            status=200, content_type='application/json',
+            match_querystring=True
+        )
+        # mock patch subscription request
+        responses.add(
+            responses.PATCH,
+            'http://localhost:8005/api/v1/subscriptions/%s/' % subscription_id,
+            json={"id": subscription_id,
```
Comment (imsickofmaps) [inline changes/tests.py]: Remove the non-PATCHable items please

File: hellomama_registration/utils.py [hunk 2]
```diff
@@ -63,6 +63,47 @@ def get_schedule(schedule_id):
...
+    data = data
+    headers = {'Authorization': [
+        'Token %s' % settings.STAGE_BASED_MESSAGING_TOKEN],
+        'Content-Type': ['application/json']
+    }
+    r = requests.patch(url, data=data, headers=headers)
+    return r.json()
+
+
+def deactivate_subscription(subscription):
```
Comment (imsickofmaps) [inline hellomama_registration/utils.py:93]: Want a mop?

**Comment by imsickofmaps** [comment]
Minor changes then :+1: when travis happy

**Pull request merged**